### PR TITLE
Pioneer 20c glider cals

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: calibration/NUTNRM/CGINS-NUTNRM-00728__20190403.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...3.19, 273.99, 274.8, 275.61, 276.41, 277.22, 278.03, 278.83, 279.64, 280.45, 281.26, 282.06, 282.87, 283.68, 284.48, 285.29, 286.1, 286.91, 287.71, 288.52, 289.33, 290.13, 290.94, 291.75, 292.56, 293.36, 294.17, 294.98, 295.79, 296.59, 297.4, 298.21, 299.01, 299.82, 300.63, 301.43, 302.24, 303.05, 303.86, 304.66, 305.47, 306.28, 307.08, 307.89, 308.7, 309.5, 310.31, 311.12, 311.92, 312.73, 313.54, 314.34, 315.15, 315.95, 316.76, 317.57, 318.37, 319.18, 319.98, 320.79, 321.6, 322.4, 323.21, 324.01, 324.82, 325.62, 326.43, 327.23, 328.04, 328.84, 329.65, 330.45, 331.26, 332.06, 332.87, 333.67, 334.48, 335.28, 336.08, 336.89, 337.69, 338.5, 339.3, 340.1, 340.91, 341.71, 342.51, 343.32, 344.12, 344.92, 345.72, 346.53, 347.33, 348.13, 348.93, 349.74, 350.54, 351.34, 352.14, 352.94, 353.74, 354.55, 355.35, 356.15, 356.95, 357.75, 358.55, 359.35, 360.15, 360.95, 361.75, 362.55, 363.35, 364.15, 364.95, 365.74, 366.54, 367.34, 368.14, 368.94, 369.74, 370.53, 371.33, 372.13, 372.93, 373.72, 374.52, 375.32, 376.11, 376.91, 377.71, 378.5, 379.3, 380.09, 380.89, 381.68, 382.48, 383.27, 384.07, 384.86, 385.66, 386.45, 387.24, 388.04, 388.83, 389.62, 390.41, 391.21, 392, 392.79, 393.58, 394.37]",
```
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00728__20190403.csv:8]: spot checked, looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00728__20240808.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...3.19, 273.99, 274.8, 275.61, 276.41, 277.22, 278.03, 278.83, 279.64, 280.45, 281.26, 282.06, 282.87, 283.68, 284.48, 285.29, 286.1, 286.91, 287.71, 288.52, 289.33, 290.13, 290.94, 291.75, 292.56, 293.36, 294.17, 294.98, 295.79, 296.59, 297.4, 298.21, 299.01, 299.82, 300.63, 301.43, 302.24, 303.05, 303.86, 304.66, 305.47, 306.28, 307.08, 307.89, 308.7, 309.5, 310.31, 311.12, 311.92, 312.73, 313.54, 314.34, 315.15, 315.95, 316.76, 317.57, 318.37, 319.18, 319.98, 320.79, 321.6, 322.4, 323.21, 324.01, 324.82, 325.62, 326.43, 327.23, 328.04, 328.84, 329.65, 330.45, 331.26, 332.06, 332.87, 333.67, 334.48, 335.28, 336.08, 336.89, 337.69, 338.5, 339.3, 340.1, 340.91, 341.71, 342.51, 343.32, 344.12, 344.92, 345.72, 346.53, 347.33, 348.13, 348.93, 349.74, 350.54, 351.34, 352.14, 352.94, 353.74, 354.55, 355.35, 356.15, 356.95, 357.75, 358.55, 359.35, 360.15, 360.95, 361.75, 362.55, 363.35, 364.15, 364.95, 365.74, 366.54, 367.34, 368.14, 368.94, 369.74, 370.53, 371.33, 372.13, 372.93, 373.72, 374.52, 375.32, 376.11, 376.91, 377.71, 378.5, 379.3, 380.09, 380.89, 381.68, 382.48, 383.27, 384.07, 384.86, 385.66, 386.45, 387.24, 388.04, 388.83, 389.62, 390.41, 391.21, 392, 392.79, 393.58, 394.37]",
```
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00728__20240808.csv:8]: spot checked, looks good

**Review by cdobs — COMMENTED** [review]
First approval for both NUTNR cals only

File: calibration/CTDGVM/CGINS-CTDGVM-09338__20220823.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9338,CC_calibration_date,1661212800000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09338__20220823.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00499__20221005.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+499,CC_calibration_date,1664928000000,sensor calibration date (milliseconds since 1970-01-01)
+499,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00499__20221005.csv:3]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00543__20210927.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+543,CC_calibration_date,1632700800000,sensor calibration date (milliseconds since 1970-01-01)
+543,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00543__20210927.csv:3]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00543__20221214.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+543,CC_calibration_date,1670976000000,sensor calibration date (milliseconds since 1970-01-01)
+543,CC_ref_salinity,0,Optode salinity setting
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00543__20221214.csv:3]: looks good

File: calibration/FLORTM/CGINS-FLORTM-04472__20220805.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4472,CC_angular_resolution,1.076,
+4472,CC_depolarization_ratio,0.039,
+4472,CC_measurement_wavelength,700,
+4472,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04472__20220805.csv:5]: looks good

File: calibration/PARADM/CGINS-PARADM-50242__20220812.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50242,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50242__20220812.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00499__20221005.csv
Comment (reedan88) [inline calibration/DOSTAM/CGINS-DOSTAM-00499__20221005.csv:1]: Looks good.

**Pull request merged**